graphbolt/include/graphbolt/threading_utils.h

```diff
@@ -0,0 +1,30 @@
+/**
+ *  Copyright (c) 2023 by Contributors
+ * @file graphbolt/utils.h
+ * @brief Threading utils in graphbolt.
+ */
+#ifndef GRAPHBOLT_THREADING_UTILS_H_
+#define GRAPHBOLT_THREADING_UTILS_H_
+
+#include <omp.h>
+
+namespace graphbolt {
+namespace runtime {
```

Comment:
as we're adding more and more files in this path, I think we need to split them into further sub-directories, especially the ones lie in different namespaces. for this one, it's in namespace `runtime` while some of other existing files are in namespace `sampling`.